src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = props => (
```

Comment:
better to destruct props
```suggestion
export const Sum =({a, b}) => (
```